tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitThis": true,
-    "alwaysStrict": true
+    "alwaysStrict": true,
+    "outDir": "dist"
```

Comment:
`outDir` is already set for transpiling: https://github.com/jsynowiec/node-typescript-boilerplate/blob/master/tsconfig.release.json#L7 with a few additional *release* options.

`tsconfig.json` is supposed to be used by tsserver only for linting and validating sources (e.g. editor integration).